### PR TITLE
[imx] Update imx SDK to 6.1 langdale with reduced SDK size

### PR DESCRIPTION
**Pull request by TE-N-ElvenWang**
Update imx SDK to 6.1 langdale with reduced SDK size since the container has disk space limitation.

### CODE REVIEW

File: integrations/docker/images/chip-build-imx/Dockerfile [hunk 1]
```diff
@@ -22,6 +22,7 @@ RUN set -x \
 
 FROM connectedhomeip/chip-build:${VERSION}
 
+
 COPY --from=build /opt/fsl-imx-xwayland /opt/fsl-imx-xwayland
 
-ENV IMX_SDK_ROOT=/opt/fsl-imx-xwayland/5.15-kirkstone/
+ENV IMX_SDK_ROOT=/opt/fsl-imx-xwayland/6.1-langdale/
```
Comment (andy31415) [inline integrations/docker/images/chip-build-imx/Dockerfile:28]: you need to change `integrations/docker/images/chip-build-vscode/Dockerfile` as well (both the copy and env)

Comment (woody-apple) [reply integrations/docker/images/chip-build-imx/Dockerfile:28]: @andy31415 I think I got it?

**Review by woody-apple — APPROVED** [review]

File: integrations/docker/images/chip-build-imx/Dockerfile (hunk 1 above)
Comment (TE-N-ElvenWang) [reply integrations/docker/images/chip-build-imx/Dockerfile:28]: > you need to change `integrations/docker/images/chip-build-vscode/Dockerfile` as well (both the copy and env)

Thank you for the comments, it is fixed in the updated PR.

File: integrations/docker/images/chip-build-vscode/Dockerfile [hunk 2]
```diff
@@ -97,7 +97,7 @@ ENV CY_TOOLS_PATHS="/opt/ModusToolbox/tools_2.4"
 ENV SILABS_BOARD=BRD4161A
 ENV IDF_PATH=/opt/espressif/esp-idf/
 ENV IDF_TOOLS_PATH=/opt/espressif/tools
-ENV IMX_SDK_ROOT=/opt/fsl-imx-xwayland/5.15-kirkstone/
+ENV IMX_SDK_ROOT=/opt/fsl-imx-internal-xwayland/6.1-langdale
```
Comment (andy31415) [inline integrations/docker/images/chip-build-vscode/Dockerfile:100]: what is the `internal` part? Should this not be the same as the chip-build-imx part?

Also line 49 above seems to have the copy without internal.

File: integrations/docker/images/chip-build/version [hunk 3]
```diff
@@ -1 +1 @@
-0.7.19.1 Version bump reason: Reducing size of imx
+0.7.19.2 Version bump reason: Update imx SDK
```
[the remark this answers is not in the document]
Comment (TE-N-ElvenWang) [reply integrations/docker/images/chip-build/version]: Thanks for the comments, dropped the forth digit and keep 0.7.20 only.

File: integrations/docker/images/chip-build-vscode/Dockerfile (hunk 2 above)
Comment (TE-N-ElvenWang) [reply integrations/docker/images/chip-build-vscode/Dockerfile]: ah, this is my mistake, I'm apologize... thank you for pointed this!
corrected in the updated PR.

**Review by andy31415 — APPROVED** [review]

**Pull request merged**